### PR TITLE
Fee breakdown copies

### PR DESCRIPTION
**Pull request by raf-rdx**

### CODE REVIEW

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 1]
```diff
@@ -18,6 +18,21 @@
 	"transactionReview_networkFee_heading": "Transaction Fee",
 	"transactionReview_networkFee_congestedText": "The network is currently congested. Add a tip to speed up your transfer.",
 	"transactionReview_networkFee_customizeButtonTitle": "Customize",
+	"transactionReview_networkFeeSheet_default_title": "Customize Fees",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:21]: I think we should use `customizeNetworkFeeSheet`

Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:21]: I would change to `default` to `normalMode`
```suggestion
	"transactionReview_networkFeeSheet_normalMode_title": "Customize Fees",
```

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 2]
```diff
@@ -18,6 +18,21 @@
...
+	"transactionReview_networkFeeSheet_advanced_title": "Advanced Customize Fees",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:22]:
```suggestion
	"transactionReview_networkFeeSheet_advancedMode_title": "Advanced Customize Fees",
```

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 3]
```diff
@@ -18,6 +18,21 @@
...
+	"transactionReview_networkFeeSheet_default_subtitle": "Choose what account to pay the transaction fee from, or add a “tip” to speed up your transaction if necessary.",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:23]:
```suggestion
	"transactionReview_networkFeeSheet_normalMode_subtitle": "Choose what account to pay the transaction fee from, or add a “tip” to speed up your transaction if necessary.",
```

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 4]
```diff
@@ -18,6 +18,21 @@
...
+	"transactionReview_networkFeeSheet_advanced_subtitle": "Fully customize fee payment for this transaction. Not recommended unless you are a developer or advanced user.",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:24]:
```suggestion
	"transactionReview_networkFeeSheet_advancedMode_subtitle": "Fully customize fee payment for this transaction. Not recommended unless you are a developer or advanced user.",
```

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 5]
```diff
@@ -18,6 +18,21 @@
...
+	"transactionReview_networkFeeSheet_payFeeFrom": "Pay fee from",
+	"transactionReview_networkFeeSheet_changeButtonTitle": "Change",
+	"transactionReview_networkFeeSheet_noneRequired": "None required",
+	"transactionReview_networkFeeSheet_noneDue": "None due",
+	"transactionReview_networkFeeSheet_selectFeePayerButtonTitle": "Select Fee Payer",
+	"transactionReview_networkFeeSheet_viewNormalModeButtonTitle": "View Normal Mode",
+	"transactionReview_networkFeeSheet_viewAdvancedModeButtonTitle": "View Advanced Mode",
+	"transactionReview_networkFeeSheet_networkFee": "Network Fee",
+	"transactionReview_networkFeeSheet_royaltyFees": "Royalty fees",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:33]: `Fee` should not be plural, no?
```suggestion
	"transactionReview_networkFeeSheet_royaltyFee": "Royalty Fee",
```

File: Content/1.1 - Transaction Review/transactionReview.json [hunk 6]
```diff
@@ -18,6 +18,21 @@
...
+	"transactionReview_networkFeeSheet_networkRoyaltyFeesFieldLabel": "XRD to Lock for Network and Loyalty Fees",
```
Comment (GhenadieVP) [inline Content/1.1 - Transaction Review/transactionReview.json:34]: `Loyalty` is also nice, but we should have `Royalty` :)
```suggestion
	"transactionReview_networkFeeSheet_networkRoyaltyFeesFieldLabel": "XRD to Lock for Network and Royalty Fees",
```

**Review by GhenadieVP — COMMENTED** [review]
LGTM,
Added some suggestions, and some fixes that are needed

**Comment by GhenadieVP** [comment]
Almost forgot, we do also need warning and error messages:
- Warning instructing users to select a fee payer.
- Error when user selected account with insufficient balance.

**Comment by raf-rdx** [comment]
> Almost forgot, we do also need warning and error messages:
> 
> * Warning instructing users to select a fee payer.
> * Error when user selected account with insufficient balance.

I have added these, let me know if its okay:
```
"transactionReview_customizeNetworkFeeSheet_selectFeePayer_warning": "Please select a fee payer for the transaction fee",
	"transactionReview_customizeNetworkFeeSheet_insufficientBalance_warning": "Insufficient balance to pay the transaction fee",
```

**Comment by GhenadieVP** [comment]
I actually think we need one more string - In choose fee payer screen, we should also have string of sorts - "Select an account to pay %s XRD transaction fee". It will probably be a subtitle to "Select Fee Payer" title

**Comment by GhenadieVP** [comment]
@raf-rdx I don't think we will get design for that, and thought we have semi-agreed on what we should try to do.

**Comment by raf-rdx** [comment]
> @raf-rdx I don't think we will get design for that, and thought we have semi-agreed on what we should try to do.

@GhenadieVP Ok I just don't recall to have this additional string only the title (Select Fee Payer), so if you prefer to have it, then I can add it.
Additionally on Android I think we will keep list of accounts simple i.e. without balance as we said on demo to keep it simple, so that's why we might not use it. But in case we change mind, then we may make a use of that 👍

**Comment by raf-rdx** [comment]
> > @raf-rdx I don't think we will get design for that, and thought we have semi-agreed on what we should try to do.
> 
> @GhenadieVP Ok I just don't recall to have this additional string only the title (Select Fee Payer), so if you prefer to have it, then I can add it. Additionally on Android I think we will keep list of accounts simple i.e. without balance as we said on demo to keep it simple, so that's why we might not use it. But in case we change mind, then we may make a use of that 👍

@GhenadieVP Is the naming correct for you? :
"transactionReview_customizeNetworkFeeSheet_selectFeePayer_subtitle": "Select an account to pay %s XRD transaction fee" ?

**Review by GhenadieVP — APPROVED** [review]

**Pull request merged**